packages/react-vapor/src/components/color/Color.tsx

```diff
@@ -24,9 +24,7 @@ export class Color extends React.Component<ColorProps & React.HTMLProps<HTMLDivE
             <div
                 {..._.omit(this.props, 'color', 'hex')}
                 style={colorStyle}
-                className={classNames(this.props.className, {
-                    [`bg-${this.props.color}`]: !isHex,
-                })}
+                className={classNames(this.props.className)}

```

Comment:
if this.props.className is a string you can do
```suggestion
                className={this.props.className}
```

Reply:
the defaultProps should change from pure-white to `#fff` at line 14

I also think we should remove the isHex and simply considering that the props.color is always a hex

Reply:
#FFF is basically the same as #FFFFFF right?  :thinking:

Reply:
Yes